src/interface_elements.jl

```diff
@@ -0,0 +1,449 @@
+                             gip::Union{InterfaceCellInterpolation, VectorizedInterpolation{<:Any,<:Any,<:Any,<:InterfaceCellInterpolation}} = default_geometric_interpolation(ip))
+    return InterfaceCellValues(Float64, qr, ip, gip)
+end
+
+function InterfaceCellValues(::Type{T}, qr::QuadratureRule, ip::InterfaceCellInterpolation) where {T}
+    return InterfaceCellValues(T, qr, ip, default_geometric_interpolation(ip))
+end
+function InterfaceCellValues(::Type{T}, qr::QuadratureRule, ip::VectorizedInterpolation{<:Any,<:Any,<:Any,<:InterfaceCellInterpolation}) where {T}
+    return InterfaceCellValues(T, qr, ip, default_geometric_interpolation(ip.ip))
+end
+
+function InterfaceCellValues(::Type{T}, qr::QuadratureRule, 
+                             ip::Union{InterfaceCellInterpolation, VectorizedInterpolation{<:Any,<:Any,<:Any,<:InterfaceCellInterpolation}}, 
+                             sgip::InterfaceCellInterpolation{sdim}) where {T, sdim}
+    return InterfaceCellValues(T, qr, ip, sgip^sdim)
+end
+
+function InterfaceCellValues(::Type{T}, qr::QR, ip::IP, gip::VGIP) where {
+    T, sdim, rdim, shape <: AbstractRefShape{sdim}, rshape <: AbstractRefShape{rdim},
+    QR  <: QuadratureRule{rshape},
+    IPhere  <: ScalarInterpolation{rshape},
+    IPthere  <: ScalarInterpolation{rshape},
+    IP <: InterfaceCellInterpolation{sdim, rdim, shape, rshape, IPhere, IPthere},
+    GIPhere <:ScalarInterpolation{rshape},
+    GIPthere <:ScalarInterpolation{rshape},
+    GIP <: InterfaceCellInterpolation{sdim, rdim, shape, rshape, GIPhere, GIPthere},
+    VGIP <: VectorizedInterpolation{sdim, shape, <:Any, GIP}
+}
+    here = CellValues(T, qr, ip.here, gip.ip.here^sdim)
+    there = CellValues(T, qr, ip.there, gip.ip.there^sdim)
+    return InterfaceCellValues(ip, here, there)
+end
+
+function InterfaceCellValues(::Type{T}, qr::QR, ip::VIP, gip::VGIP) where {
+    T, sdim, vdim, rdim, shape <: AbstractRefShape{sdim}, rshape <: AbstractRefShape{rdim},
+    QR  <: QuadratureRule{rshape},
+    IPhere  <: ScalarInterpolation{rshape},
+    IPthere  <: ScalarInterpolation{rshape},
+    IP <: InterfaceCellInterpolation{sdim, rdim, shape, rshape, IPhere, IPthere},
+    VIP <: VectorizedInterpolation{vdim, shape, <:Any, IP},
+    GIPhere <:ScalarInterpolation{rshape},
+    GIPthere <:ScalarInterpolation{rshape},
+    GIP <: InterfaceCellInterpolation{sdim, rdim, shape, rshape, GIPhere, GIPthere},
+    VGIP <: VectorizedInterpolation{sdim, shape, <:Any, GIP}
+}
+    here = CellValues(T, qr, ip.ip.here^vdim, gip.ip.here^sdim)
+    there = CellValues(T, qr, ip.ip.there^vdim, gip.ip.there^sdim)
+    return InterfaceCellValues(ip, here, there)
+end
+
+reinit!(cv::InterfaceCellValues, cc::CellCache) = reinit!(cv, cc.coords)
+
+function reinit!(cv::InterfaceCellValues, x::AbstractVector{Vec{sdim,T}}) where {sdim, T}
+    nsplit = getngeobasefunctions(cv.here)
+    reinit!(cv.here, x[1:nsplit])
+    reinit!(cv.there, x[nsplit+1:end])
```

Comment:
```suggestion
    reinit!(cv.here, @view(x[1:nsplit]))
    reinit!(cv.there, @view(x[nsplit+1:end]))
```

Reply:
Just for my understanding:
This would avoid allocations, right?
(Or what would be the benefit of your suggestion?)